packages/styles/src/components/accordion.scss

```diff
@@ -1,94 +1,103 @@
 @forward './../variables/options';
 
-@use './../lic/bootstrap-license';
-@use './../themes/bootstrap/core' as *;
-@use './../themes/bootstrap/accordion' as ba;
+@use './../mixins/button' as button-mixins;
+@use './../mixins/color' as color-mixins;
+@use './../mixins/icons' as icon-mixins;
+@use './../mixins/type' as type-mixins;
+@use './../mixins/utilities' as utility-mixins;
 
-@use './../mixins/type';
-@use './../mixins/utilities';
-@use './../variables/spacing';
 @use './../variables/components/accordion';
-@use './../variables/components/button';
 
 .accordion-item {
-  border-right: 0;
-  border-left: 0;
-  background-color: transparent;
+  border-block-start: accordion.$accordion-border-width solid accordion.$accordion-border-color;
+}
 
-  &:last-of-type {
-    border-bottom: 0;
-  }
+.accordion-header {
+  @include type-mixins.font-curve(accordion.$accordion-header-font-curve);
+  font-weight: accordion.$accordion-header-font-weight;
+  line-height: accordion.$accordion-header-line-height;
+  margin: 0;
```

Comment:
CSS is awesome

<img width="1360" alt="image" src="https://github.com/swisspost/design-system/assets/1659006/016b28ef-9f0c-443b-8604-d3c9577760f7">

For the text-truncate class to work, the span has to be `display: inline-block` (leaves that invisible margin-bottom or `display: block`. I think the use-case here, because it's documented, warrants a dedicated workaround where we specifically style set the span with class text-truncate to display: block. What do you think?

Reply:
I added a `display: flex;` on the accordion button so that we don't have to style unknown children

Reply:
I don't think that worked, looks like this for me:

<img width="1066" alt="image" src="https://github.com/swisspost/design-system/assets/1659006/34a9a04c-4d6a-4f96-85eb-63010b216d87">

Now the fancy title is wrapping and the overflow is still there.

Reply:
My bad, the truncated title work in Firefox but I did not see the "Fancy title"... 
I just updated the PR to give a `display: block` to `.text-truncate` elements.